[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:00:29,618 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:00:29,718 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:00:30,631 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:00:30,721 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:01:00,721 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:01:00,821 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:01:01,730 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:01:01,835 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:01:31,824 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:01:31,924 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:01:32,835 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:01:32,926 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:02:02,927 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:02:03,027 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:02:03,938 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:02:04,029 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:02:34,029 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:02:34,129 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:02:35,041 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:02:35,150 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:03:05,132 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:03:05,232 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:03:06,134 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:03:06,234 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:03:36,235 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:03:36,335 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:03:37,241 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:03:37,347 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:04:07,338 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:04:07,438 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:04:08,341 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:04:08,441 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:04:38,440 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:04:38,540 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:04:39,442 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:04:39,542 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:05:09,660 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:05:09,662 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:05:10,715 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:05:11,065 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:05:40,747 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:05:41,146 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:05:41,747 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:05:42,147 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:06:11,849 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:06:12,249 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:06:12,868 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:06:13,250 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:06:42,951 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:06:43,351 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:06:43,953 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:06:44,353 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:07:14,055 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:07:14,455 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:07:15,060 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:07:15,463 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:07:45,157 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:07:45,557 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:07:46,171 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:07:46,568 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:08:16,260 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:08:16,660 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:08:17,262 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:08:17,662 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:08:47,363 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:08:47,763 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:08:48,368 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:08:48,768 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:09:18,465 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:09:18,865 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:09:19,468 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:09:19,866 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:09:49,568 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:09:49,968 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:09:50,569 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:09:50,969 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:10:20,671 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:10:21,071 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:10:21,672 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:10:22,067 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:10:51,774 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:10:52,074 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:10:52,776 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:10:53,075 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:11:22,876 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:11:23,176 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:11:23,878 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:11:24,178 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:11:53,979 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:11:54,279 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:11:54,992 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:11:55,280 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:12:25,083 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:12:25,382 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:12:26,095 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:12:26,393 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:12:56,191 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:12:56,485 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:12:57,206 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:12:57,486 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:13:27,287 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:13:27,587 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:13:28,290 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:13:28,590 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:13:58,390 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:13:58,690 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:13:59,397 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:13:59,698 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:14:29,493 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:14:29,793 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:14:30,496 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:14:30,806 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:15:00,596 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:15:00,896 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:15:01,590 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:15:01,907 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:15:31,599 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:15:31,999 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:15:32,631 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:15:33,031 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:16:02,701 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:16:03,101 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:16:03,704 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:16:04,100 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:16:33,804 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:16:34,104 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:16:34,807 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:16:35,117 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:17:04,907 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:17:05,207 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:17:05,911 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:17:06,201 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:17:36,009 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:17:36,211 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:17:37,003 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:17:37,223 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:18:07,012 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:18:07,312 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:18:08,023 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:18:08,315 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:18:38,116 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:18:38,416 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:18:39,127 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:18:39,417 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:19:09,218 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:19:09,518 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:19:10,286 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:19:10,525 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:19:40,321 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:19:40,621 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:19:41,341 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:19:41,641 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:20:11,423 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:20:11,723 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:20:12,425 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:20:12,719 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:20:42,526 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:20:42,726 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:20:43,528 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:20:43,729 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:21:13,629 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:21:13,829 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:21:14,630 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:21:14,829 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:21:44,732 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:21:44,932 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:21:45,733 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:21:45,933 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:22:15,834 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:22:16,034 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:22:16,835 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:22:17,045 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:22:46,938 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:22:47,138 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:22:47,970 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:22:48,170 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:23:18,040 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:23:18,240 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:23:19,048 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:23:19,247 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:23:49,143 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:23:49,343 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:23:50,167 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:23:50,349 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:24:20,245 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:24:20,445 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:24:21,259 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:24:21,459 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:24:51,348 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:24:51,548 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:24:52,344 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:24:52,549 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:25:22,351 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:25:22,651 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:25:23,354 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:25:23,654 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:25:53,454 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:25:53,754 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:25:54,459 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:25:54,758 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:26:24,556 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:26:24,856 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:26:25,558 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:26:25,857 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:26:55,660 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:26:55,960 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:26:56,661 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:26:56,971 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:27:26,762 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:27:27,062 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:27:27,764 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:27:28,074 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:27:57,865 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:27:58,165 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:27:58,946 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:27:59,246 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:28:28,967 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:28:29,267 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:28:30,005 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:28:30,305 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:29:00,070 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:29:00,370 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:29:01,072 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:29:01,370 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:29:31,173 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:29:31,373 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:29:32,175 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:29:32,374 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:30:02,276 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:30:02,476 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:30:03,277 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:30:03,477 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:30:33,378 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:30:33,578 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:30:34,379 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:30:34,589 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:31:04,481 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:31:04,681 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:31:05,483 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:31:05,697 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:31:35,584 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:31:35,784 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:31:36,606 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:31:36,797 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:32:06,692 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:32:06,888 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:32:07,799 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:32:07,889 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:32:37,890 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:32:37,990 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:32:38,888 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:32:38,988 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:33:08,893 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:33:08,993 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:33:09,958 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:33:10,005 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:33:39,995 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:33:40,095 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:33:41,007 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:33:41,100 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:34:11,098 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:34:11,198 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:34:12,108 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:34:12,198 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:34:42,201 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:34:42,301 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:34:43,210 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:34:43,300 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:35:13,303 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:35:13,303 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:35:14,304 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:35:14,306 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:35:44,406 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:35:44,406 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:35:45,419 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:35:45,454 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:36:15,510 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:36:15,509 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:36:16,518 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:36:16,524 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:36:46,612 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:36:46,612 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:36:47,621 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:36:47,623 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:37:17,715 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:37:17,717 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:37:18,723 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:37:18,733 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:37:48,817 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:37:48,817 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:37:49,832 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:37:49,837 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:38:19,921 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:38:19,920 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:38:20,927 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:38:20,930 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:38:51,023 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:38:51,023 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:38:52,072 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:38:52,101 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:39:22,126 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:39:22,126 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:39:23,138 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:39:23,167 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:39:53,228 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:39:53,228 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:39:54,239 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:39:54,240 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:40:24,331 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:40:24,331 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:40:25,337 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:40:25,338 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:40:55,434 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:40:55,434 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:40:56,438 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:40:56,444 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:41:26,537 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:41:26,537 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:41:27,542 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:41:27,546 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:41:57,642 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:41:57,640 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:41:58,646 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:41:58,648 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:42:28,742 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:42:28,742 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:42:29,755 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:42:29,755 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:42:59,845 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:42:59,845 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:43:00,847 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:43:00,847 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:43:30,948 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:43:30,948 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:43:31,957 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:43:31,966 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:44:02,050 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:44:02,051 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:44:03,045 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:44:03,051 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:44:33,053 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:44:33,153 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:44:34,059 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:44:34,165 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:45:04,156 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:45:04,256 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:45:05,160 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:45:05,260 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:45:35,259 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:45:35,359 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:45:36,269 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:45:36,379 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:46:06,362 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:46:06,462 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:46:07,355 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:46:07,552 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:46:37,364 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:46:37,564 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:46:38,358 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:46:38,565 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:47:08,367 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:47:08,667 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:47:09,369 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:47:09,667 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:47:39,470 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:47:39,770 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:47:40,480 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:47:40,775 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:48:09,793 INFO [http-nio-56085-exec-4][QCloudSmsService.java:56] - 给手机号13344334433发送验证码：108163
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:48:10,573 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:48:10,873 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:48:11,580 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:48:11,891 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:48:20,764 ERROR [http-nio-56085-exec-4][GlobalExceptionHandler.java:45] - 【系统异常】{}
org.springframework.dao.QueryTimeoutException: Redis command timed out; nested exception is io.lettuce.core.RedisCommandTimeoutException: Command timed out after 10 second(s)
	at org.springframework.data.redis.connection.lettuce.LettuceExceptionConverter.convert(LettuceExceptionConverter.java:70)
	at org.springframework.data.redis.connection.lettuce.LettuceExceptionConverter.convert(LettuceExceptionConverter.java:41)
	at org.springframework.data.redis.PassThroughExceptionTranslationStrategy.translate(PassThroughExceptionTranslationStrategy.java:44)
	at org.springframework.data.redis.FallbackExceptionTranslationStrategy.translate(FallbackExceptionTranslationStrategy.java:42)
	at org.springframework.data.redis.connection.lettuce.LettuceConnection.convertLettuceAccessException(LettuceConnection.java:268)
	at org.springframework.data.redis.connection.lettuce.LettuceStringCommands.convertLettuceAccessException(LettuceStringCommands.java:799)
	at org.springframework.data.redis.connection.lettuce.LettuceStringCommands.setEx(LettuceStringCommands.java:232)
	at org.springframework.data.redis.connection.DefaultedRedisConnection.setEx(DefaultedRedisConnection.java:295)
	at org.springframework.data.redis.connection.DefaultStringRedisConnection.setEx(DefaultStringRedisConnection.java:982)
	at org.springframework.data.redis.core.DefaultValueOperations$4.potentiallyUsePsetEx(DefaultValueOperations.java:268)
	at org.springframework.data.redis.core.DefaultValueOperations$4.doInRedis(DefaultValueOperations.java:261)
	at org.springframework.data.redis.core.RedisTemplate.execute(RedisTemplate.java:224)
	at org.springframework.data.redis.core.RedisTemplate.execute(RedisTemplate.java:184)
	at org.springframework.data.redis.core.AbstractOperations.execute(AbstractOperations.java:95)
	at org.springframework.data.redis.core.DefaultValueOperations.set(DefaultValueOperations.java:256)
	at cn.itcast.sailing.common.cache.RedisCache.set(RedisCache.java:44)
	at cn.itcast.sailing.store.RedisVerificationStore.set(RedisVerificationStore.java:15)
	at cn.itcast.sailing.handler.AbstractVerificationHandler.generateVerificationInfo(AbstractVerificationHandler.java:54)
	at cn.itcast.sailing.service.VerificationService.generateVerificationInfo(VerificationService.java:28)
	at cn.itcast.sailing.controller.VerificationController.generateVerificationInfo(VerificationController.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:189)
	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:138)
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:102)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:895)
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:800)
	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87)
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1038)
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:942)
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1005)
	at org.springframework.web.servlet.FrameworkServlet.doPost(FrameworkServlet.java:908)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:660)
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:882)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.springframework.boot.actuate.web.trace.servlet.HttpTraceFilter.doFilterInternal(HttpTraceFilter.java:90)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:99)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:92)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.springframework.web.filter.HiddenHttpMethodFilter.doFilterInternal(HiddenHttpMethodFilter.java:93)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.springframework.boot.actuate.metrics.web.servlet.WebMvcMetricsFilter.filterAndRecordMetrics(WebMvcMetricsFilter.java:117)
	at org.springframework.boot.actuate.metrics.web.servlet.WebMvcMetricsFilter.doFilterInternal(WebMvcMetricsFilter.java:106)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:200)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:200)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:490)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:408)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:834)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1415)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
Caused by: io.lettuce.core.RedisCommandTimeoutException: Command timed out after 10 second(s)
	at io.lettuce.core.ExceptionFactory.createTimeoutException(ExceptionFactory.java:51)
	at io.lettuce.core.LettuceFutures.awaitOrCancel(LettuceFutures.java:114)
	at io.lettuce.core.FutureSyncInvocationHandler.handleInvocation(FutureSyncInvocationHandler.java:69)
	at io.lettuce.core.internal.AbstractInvocationHandler.invoke(AbstractInvocationHandler.java:80)
	at com.sun.proxy.$Proxy134.setex(Unknown Source)
	at org.springframework.data.redis.connection.lettuce.LettuceStringCommands.setEx(LettuceStringCommands.java:230)
	... 76 more
[${project.name}][${env:SERVER_PORT}] 2019-12-28 11:48:41,677 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
